ci/scenario_helpers/ci_constants.py

```diff
@@ -72,17 +76,19 @@ class CIConstants(object):
 
     HA_TIMEOUT = 300
 
+    api = OVSClient(SETUP_CFG['ci']['grid_ip'], SETUP_CFG['ci']['user']['api']['username'], SETUP_CFG['ci']['user']['api']['password'])
```

Comment:
Change this to:
```
@property
def api(self)...
```
Or when we do not use an instance (which is everywhere for this CIConstants
You can introduce a new decorator:
```
class classproperty(property):
    def __get__(self, cls, owner):
        return classmethod(self.fget).__get__(None, owner)()
```
And use it like
```
class Test(object):
    _my_hidden_prop = 'I was hidden'
    @classproperty
    def test_prop(cls):
        return '{0}. Now revealed!'.format(cls._my_hidden_prop)
```

Reply:
Very useful, thanks